crud.py

```diff
@@ -46,6 +48,10 @@ async def create_market_product(data: createProduct) -> Products:
     )
     product = await get_market_product(product_id)
     assert product, "Newly created product couldn't be retrieved"
+    stall = await get_market_stall(product.stall)
```

Comment:
- duplicate logic?
- `get_market_stall()` already does the `product.price` computation